source/unified-test-format/schema-1.14.json

```diff
@@ -0,0 +1,682 @@
+            "additionalProperties": false,
+            "properties": {}
+          },
+          "connectionCreatedEvent": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {}
+          },
+          "connectionReadyEvent": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {}
+          },
+          "connectionClosedEvent": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+              "reason": { "type": "string" }
+            }
+          },
+          "connectionCheckOutStartedEvent": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {}
+          },
+          "connectionCheckOutFailedEvent": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+              "reason": { "type": "string" }
+            }
+          },
+          "connectionCheckedOutEvent": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {}
+          },
+          "connectionCheckedInEvent": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {}
+          }
+        }
+      },
+  
+      "expectedSdamEvent": {
+        "type": "object",
+        "additionalProperties": false,
+        "maxProperties": 1,
+        "minProperties": 1,
+        "properties": {
+          "serverDescriptionChangedEvent": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+              "previousDescription": { "$ref": "#/definitions/serverDescription" },
+              "newDescription": { "$ref": "#/definitions/serverDescription" }
+            }
+          }
+        }
+      },
+  
+      "serverDescription": {
+        "type": "object",
+        "additionalProperties": false,
+        "properties": {
+          "type": { 
+            "type": "string",
+            "enum": [
+              "Standalone",
+              "Mongos",
+              "PossiblePrimary",
+              "RSPrimary",
+              "RSSecondary",
+              "RSOther",
+              "RSArbiter",
+              "RSGhost",
+              "LoadBalancer",
+              "Unknown"
+            ]
+          }
+        }
+      },
+
+      "expectedLogMessagesForClient": {
+        "type": "object",
+        "additionalProperties": false,
+        "required": ["client", "messages"],
+        "properties": {
+          "client": { "type": "string" },
+          "messages": {
+            "type": "array",
+            "items": { "$ref": "#/definitions/expectedLogMessage" } 
+            },
+          "ignoreExtraLogs": { "type": "boolean" },
+          "ignoreLogMessages": {
+            "type": "array",
+            "items": { "$ref": "#/definitions/expectedLogMessage" } 
+            }
```

Comment:
These warrant new tests under `unified-test-format/tests/invalid`. See https://github.com/mongodb/specifications/commit/d4f83d83a380a1863094c3f7bcb5d27b5963a2c9 for tests added with the original logging spec changes.

Tests for `ignoreLogMessages` will likely resemble existing tests for `messages`.

`ignoreExtraLogs` will be more straightforward and likely just need a test with an invalid type (e.g. "string" instead of boolean).

Reply:
Thanks for the pointers, done.